setup.py

```diff
@@ -1,5 +1,9 @@
+from pathlib import Path
 from setuptools import setup, find_packages
 
+this_directory = Path(__file__).parent
+version = (this_directory / "VERSION").read_text()
```

Comment:
I like this, but I would `.strip()` the resulting text. Your new `VERSION` file doesn't contain a trailing newline, but many editors will automatically add one without anyone noticing.

Reply:
That's a good call. We will have to add this to our other repos too because I copied this from them.

Reply:
gpgp. No idea if it will actually cause any issues, but it can't hurt.